src/dict.h

```diff
@@ -56,6 +56,12 @@ typedef struct dictType {
     void (*valDestructor)(dict *d, void *obj);
     int (*expandAllowed)(size_t moreMem, double usedRatio);
     void (*rehashingStarted)(dict *d);
+    size_t (*keyLen)(const void *key);
+    size_t (*keyToBytes)(unsigned char *buf, const void *key, uint8_t *header_size);
+    size_t (*valLen)(const void *val);
+    void (*valToBytes)(void *de, const void *val, unsigned char *buf);
+    int (*trySetVal)(void *de, const void *val);
```

Comment:
Still have no idea what this does, it's not actually called anywhere.

Reply:
trySetVal is a (planned) optimization for a db replace value code flow to try and fit the new value into the existing dictEntry created in the dictionary.